src/Aspect/ContractCheckerAspect.php

```diff
@@ -156,7 +164,15 @@ private function isContractSatisfied($instance, $scope, array $args, $annotation
         }
         $instance = is_object($instance) ? $instance : null;
 
-        return $invoker->bindTo($instance, $scope)->__invoke($args, $annotation->value);
+        try {
+            $invocationResult = $invoker->bindTo($instance, $scope)->__invoke($args, $annotation->value);
+        } catch (\Exception $e) {
+            throw new ContractViolation($this->invocation, $annotation->value . " Details: " . $e->getMessage());
```

Comment:
You should chain all exceptions, please add a previous exception instance as a third argument.